types/node/domain.d.ts

```diff
@@ -2,12 +2,12 @@ declare module "domain" {
     import * as events from "events";
 
     class Domain extends events.EventEmitter implements NodeJS.Domain {
-        run(fn: Function): void;
-        add(emitter: events.EventEmitter): void;
+        run<T>(fn: (...args: any[]) => T, ...args: any[]): T;
+        add(emitter: events.EventEmitter | NodeJS.Timer): void;
         remove(emitter: events.EventEmitter): void;
```

Comment:
`remove()` allows also `NodeJS.Timer` like `add()`

Reply:
Seems something got wrong during fixup as comment is marked as resolved but `remove` has not been changed.

Reply:
Please add `| NodeJS.Timer` to `remove()`